ext/random/php_random.h

```diff
@@ -37,10 +37,10 @@
 
 PHPAPI double php_combined_lcg(void);
 
-typedef struct _php_random_fallback_seed_state php_random_fallback_seed_state;
+struct _php_random_fallback_seed_state;
```

Comment:
You can keep this as is, just remove the typedef struct.. later in the same file.